sdk/src/driftClient.ts

```diff
@@ -2231,10 +2253,8 @@ export class DriftClient {
 			);
 		}
 
-		const tx = await this.buildTransaction(withdrawIxs, {
-			...(txParams ?? this.txParams),
-			computeUnits: 1_400_000,
-		});
+		const tx = await this.buildTransaction(withdrawIxs, txParams ?? this.txParams);
```

Comment:
am aware this `txParams ?? this.txParams` logic already existed but while we are refactoring `buildTransaction`, we should move that fallback logic into there